plugins/dropdown/src/mapperEvaluator.ts

```diff
@@ -0,0 +1,13 @@
+// @ts-ignore
+import SES from 'ses'
+
+const safeEvaluate = (code: String, context: any = {}) => {
+  const realm = SES.makeSESRootRealm({
+    consoleMode: 'allow'
+  })
+
+  const result = realm.evaluate(code, context)
```

Comment:
I think you had mentioned this isn't working, giving errors? I remember when I first integrated it was working and tests passing, but then my tests didn't cover all use cases

can you remind me what the error was, I think it has to do with `Array`?

Reply:
Funnily, the SES code is actually working now! On the other hand, when selecting a new selection, I am not capable of seeing the selection as selected in the dropdown.

Also, the tests for SES throw an exception

Reply:
interesting, and what is the exception? the `Array` one?

Reply:
`Uncaught Error: [mobx-state-tree] No matching type for union(null | number | string | boolean | Map...`

Reply:
oh great, that should have nothing to do with SES and be easy to fix. I'll pull down your code shortly and see if we can get that error resolved

Reply:
sent this PR to address the outstanding issues, once merged I think we can get this PR merged as well! https://github.com/fasaas/builder/pull/3